fetch/main.js

```diff
@@ -0,0 +1,13 @@
+async function pokemon() {
+  try {
+    const res = await fetch('https://pokeapi.co/api/v2/pokemon/jigglypff');
+    if (res.ok === false) {
```

Comment:
Please just use `if (!res.ok)` instead of checking to see if it matches `false`.